package.json

```diff
@@ -28,6 +28,7 @@
     "ganache:deploy": "node ganache/deploy.js"
   },
   "dependencies": {
+    "@octokit/graphql": "^4.8.0",
```

Comment:
was the other ockokit we were using removed?

Reply:
@moshmage no, we are using it on BE